### PR TITLE
Add new teams to ddqa

### PR DESCRIPTION
**Pull request by KevinFairise2**
### What does this PR do?

It adds the three new team to the ddqa config file.
It also removes Agent-Platform, that should not be used anymore for QA since we will have more precise teams

Should be merged when we start using the new labels

### CODE REVIEW

File: .ddqa/config.toml [hunk 1]
```diff
@@ -37,17 +37,41 @@ exclude_members = [
...
+  "In Progress",
+  "Done",
+]
+github_team = "agent-ci-experience"
+github_labels = ["team/agent-ci-experience"]
+exclude_members = []
+
+[teams."Agent Build and Release]
+jira_project = "BARX"
+jira_issue_type = "QA"
```
Comment (KSerrania) [inline .ddqa/config.toml:66]: @hithwen should we keep this line? It requires adding a QA issue type in the BARX Jira board.

File: .ddqa/config.toml [hunk 2]
```diff
@@ -37,17 +37,41 @@ exclude_members = [
   "cmourot",
 ]
 
-[teams."Agent Platform"]
-jira_project = "APL"
+[teams."Agent Developer Tools]
+jira_project = "ADT"
```
Comment (KSerrania) [inline .ddqa/config.toml:41]:
```suggestion
jira_project = "ADXT"
```

File: .ddqa/config.toml [hunk 3]
```diff
@@ -37,17 +37,41 @@ exclude_members = [
   "cmourot",
 ]
 
-[teams."Agent Platform"]
-jira_project = "APL"
+[teams."Agent Developer Tools]
```
Comment (KSerrania) [inline .ddqa/config.toml:40]:
```suggestion
[teams."Agent Developer Tools"]
```

File: .ddqa/config.toml [hunk 4]
```diff
@@ -37,17 +37,41 @@ exclude_members = [
...
+jira_project = "BARX"
+jira_issue_type = "QA"
+jira_statuses = [
+  "To Do",
+  "In Progress",
+  "Done",
+]
+github_team = "agent-build-and-releases"
+github_labels = ["team/agent-build-and-releases"]
+exclude_members = []
```
Comment (KSerrania) [inline .ddqa/config.toml:74]:
```suggestion
exclude_members = ["hithwen"]
```

probably

File: .ddqa/config.toml (hunk 1 above)
Comment (spencergilbert) [reply .ddqa/config.toml]: We seemed ambivalent during our sprint planning, so I've opted to swapping this to Task.

**Comment by spencergilbert** [comment]
My TOML extension on VSCode reformatted the file - happy to revert that and just do the functional changes if it's an issue.

**Review by chouetz — APPROVED** [review]

**Review by kacper-murzyn — APPROVED** [review]

**Review by KSerrania — APPROVED** [review]

**Comment by spencergilbert** [comment]
I searched for open PRs in the 7.53 milestone, with the agent-platform label and updated them to the new team labels.

This should be good to merge on Monday.

**Pull request merged**